### PR TITLE
FiBo 70 add scan receipt activity diagram

### PR DESCRIPTION
**Pull request by jens1o**

### CODE REVIEW

**Review by Jonas-Karl — CHANGES_REQUESTED** [review]
should the order of docs/use_cases/UC_6_evaluate_scanning_receipts/Slice 3.png and docs/use_cases/UC_6_evaluate_scanning_receipts/Slice 4.png change? 
It looks like slice 4 should come before slice 3.

Anyways I added some comments to clarify everything a bit more. Maybe these comments result in changing a bit more - I did not check wether or not I am destroying the feature file with the comments when you simply commit them :D

File: docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+Envisioned scanning receipts
+
+## 1.1 Brief Description
+
+Every user should be able to add data of a receipt using their camera. The data the mechanism has to automatically gather is the following:
+
+- store
+- price
+- date
```
[the remark this answers is not in the document]
Comment (ChrisEwert) [reply docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md:13]: The store can't be optional, right? It is the 'title' in our history

File: docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  Background:
+    And I am on the homepage
+
+  Scenario: open new "Scan receipt" form
+    Given I am signed in with username "USER" and password "PASSWORD"
+    And I am on the "home" page
+    When I press the "New receipt" button
+    Then I see two additional buttons "Add manually" and "Scan receipt" fade in
+    When I press the "Scan receipt" button
+    Then I am on the "scanning receipt" form
```
[the remark this answers is not in the document]
Comment (Cebox82) [reply docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md:63]: Your change is also possible, I think. But when I did another narration, I saw some examples where you use the syntax When/Then "I am"

**Review by Cebox82 — APPROVED** [review]

File: docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jens1o) [reply docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md]: Thanks, I've reflected this.

File: docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jens1o) [reply docs/use_cases/UC_6_evaluate_scanning_receipts/UC_6_evaluate_scanning_receipts.md:63]: I opted to use a "When I take a picture of a receipt"

**Pull request merged**